pages/css/architecture.md

```diff
@@ -3,36 +3,42 @@ title: Architecture
 permalink: /css/architecture/
 sidenav: css
 ---
-# Architecture
+# Choosing a CSS methodology and architecture
 A site's architecture should be based on its goals and purposes. This means the
 guidance here should be adapted to different sites and situations.
 
-## Modular or component architecture
-When using a modular or component architecture, every page is broken into a
-series of modular components. There are two sets of these components:
-`components` and `modules`. The architecture starts out with basic HTML element
-rules: HTML, p, a, form, etc tags that than have components and modules written
-on top of them. Components are very basic structure such as buttons, blurbs,
-navs, and positioning structures like insets, island, and enclosure.  From
-here, modules are built with these components. This architecture also attempts
-to keep the specificity trend in an upwards curve as you move down in the file
+In general, 18F recommends using a modular approach (specifically, [modified BEM])
+to provide consistency and scalability to larger apps. However, for some applications
+(such as a rapid prototype), an atomic architecture may make more sense. Below are 
+some considerations for choosing an architecture.
```

Comment:
I wanted to get the actionable, "we generally recommend this" bit out in front of any ideological postulation.